src/performance/server.js

```diff
@@ -88,20 +81,13 @@ app.post('/api/v1/runload', async function (req, res) {
 */
 app.post('/api/v1/cancelLoad', async function (req, res) {
     try {
-        if (req.body.url == projectURL) {
-            if (loadProcess) {
```

Comment:
Again we should probably have a check for the load process. It'll help us diagnose issues with multiple cancels or bad parameters from portal.